.circleci/config.yml

```diff
@@ -46,6 +46,10 @@ jobs:
           command: ./scripts/bundlesize
       - store_artifacts:
           path: ./bundlesize.tar
+  node10-test:
+    <<: *test
+    docker:
+      - image: chadian/nsis:10-2
```

Comment:
Before this PR can be merged we need a new oclif nsis image made for node 10. There is a [PR](https://github.com/oclif/docker/pull/1) in the `oclif/docker` repo for this.